README.md

```diff
@@ -362,21 +362,21 @@ uwsm forks a process that probes systemd activation environment for
 (whitespace-separated). When all expected vars appear, it pauses for
 `UWSM_WAIT_VARNAMES_SETTLETIME` seconds (float, default: 0.2) and signals unit
 readiness. It also updates cleanup list with delta between states of activation
-environment at unit startup time and the end of settle pause. If classic dbus
+environment at unit startup time and the end of settle pause. If classic D-Bus
 implementation is used, this delta is also synched to its activation
 environment.
 
 A separate unit, `wayland-session-waitenv.service` is launched alongside
 compositor, with similar ordering after `graphical-session-pre.target`, before
 `graphical-session.target`. It also waits for the same variables in the same
-manner, then successfully exits (or times out). Its job is to dealy
+manner, then successfully exits (or times out). Its job is to delay
 `graphical-session.target` activation in case compositor signals its readiness
 prematurely. Or to fail startup if expected vars do not appear.
 
-`uwsm finalize` command fills systemd and dbus environments with essential vars
+The `uwsm finalize` command fills systemd and D-Bus environments with essential vars
```

Comment:
I try to observe 80 char line limit in md and scd where possible, could you please reflow long lines?